### PR TITLE
[Mobile] Center Confirmation Dialog Texts

### PR DESCRIPTION
**Pull request by rukmini-bose**
Closes  https://github.com/nasa/openmct/issues/7347
### Describe your changes:

- Add code to classes that are used for _only_ confirmation dialogs such that the text and buttons for such messages are centered, rather than floating to the top. This will improve overall mobile UX. 
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Is this a breaking change to be called out in the release notes?
* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

**Comment by ozyx** [comment]
Is this visual change covered by any visual tests?

**Review by charlesh88 — CHANGES_REQUESTED** [review]
I'm finding the button centering is ending up with some unfortunate vertical alignments: 
Screenshot 2024-02-22 at 2 51 41 PM

I think it'd be better to do a right-alignment, and more consistent with the desktop layout: 

Screenshot 2024-02-22 at 2 53 13 PM

File: e2e/playwright-mobile.config.js [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 import { devices } from '@playwright/test';
 const MAX_FAILURES = 5;
-const NUM_WORKERS = 2;
```
Comment (unlikelyzero) [inline e2e/playwright-mobile.config.js:6]: driveby: reduce test flakiness

Comment (ozyx) [reply e2e/playwright-mobile.config.js:6]: nice. yeah 1 should be enough here

**Review by ozyx — APPROVED** [review]
LGTM, just one small non blocking suggestion

**Review by charlesh88 — APPROVED** [review]
After finding some newly introduced issues, have pushed up new changes to address. With that, LGTM.

**Comment by charlesh88** [comment]
Problem related to extra margin being introduced to view large/preview/properties dialogs _should_ be fixed. Visual tests should now pass...

**Review by unlikelyzero — APPROVED** [review]

**Pull request merged**